js/foundation.sticky.js

```diff
@@ -215,9 +215,6 @@ class Sticky {
                   * @event Sticky#stuckto
                   */
                  .trigger(`sticky.zf.stuckto:${stickTo}`);
-    this.$element.on("transitionend webkitTransitionEnd oTransitionEnd otransitionend MSTransitionEnd", function() {
-      _this._setSizes();
-    });
```

Comment:
It was just removed in the same commit, it's not linked. I will separate it, and include those lines deletion when i'll do the correct PR for fluid height transition (which will replace this 3 lines)